__tests__/app.test.js

```diff
@@ -409,7 +409,53 @@ test("Responds with 400 and error message if given a string instead of number fo
     .expect(400)
     .then(({ body: { msg } }) => {
       expect(msg).toBe("Incorrect type - this must be a number");
-    });
+    });S
+});
 });
+describe("GET /api/articles (queries)", () => {
+  test("Responds with 200 and articles sorted, ordered and topic chosen by the given query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author&order=ASC&topic=mitch")
+      .expect(200)
+      .then(({ body: {articles} }) => {
+        expect(articles).toBeSortedBy('author', { descending: false})
+        articles.forEach((article) => {
+          expect(article.topic).toEqual("mitch");
+        })
+        });
+      });
+      test("Responds with 404 and error message if a queries topic does not exist", () => {
```

Comment:
Un-nest these test blocks, all of the blocks below are nested in this test on line 427